usautobuild/licenser.py

```diff
@@ -41,9 +41,8 @@ def run_command(self, command: str) -> None:
         with Popen(command, stdout=PIPE, stderr=PIPE, shell=True) as cmd:
             for line, is_stdout in iterate_output(cmd):
                 if is_stdout:
-                    if line.strip():
-                        log.debug(line)
+                    log.debug(line)
                 else:
-                    if line and "Unable to find image" not in line:
+                    if "Unable to find image" not in line:
```

Comment:
I think I did the opposite before. If I found that literal, I would not raise exception as that would kill the whole process.

Reply:
original code: `line and "Unable to find image" not in line`
i just removed `line and`. are you saying original code is wrong?

Reply:
Reread the thing and I think I see what I did now. Maybe we don't need it anymore since you are pulling as part of the command?

Reply:
Problem was docker sent the "could not find local image" log to stderr so it killed the app. Then the unity process even if it failed, didn't end with status code different than zero so that was annoying and I had to read stderr instead to catch errors.

Reply:
Nah Licenser was a one time command you do to generate a license file. It doesn't run every run